packages/spatial-id-viewer/src/views/weather/view/hooks/load-models.tsx

```diff
@@ -176,15 +176,29 @@ export const createWeatherMap = (
           spatialId,
           SpatialId.fromString<WeatherInfo>(spatialId, {
             id: objectId,
-            ...definition.currentWeather,
+            startTime: definition.currentWeather.startTime,
+            endTime: definition.currentWeather.endTime,
+            'windDirection (degree)': definition.currentWeather.windDirection,
```

Comment:
I hope its not recommend to name vairables in this manner. better to rename it to windDirectionDegree